### PR TITLE
feat: Added Home and Explore page

### PR DESCRIPTION
**Pull request by mnath30**
Added the static screens for the home page and explore page
Data used is currently hardcoded which will later be replaced with the data from the backend

Preview: https://deploy-preview-1--share-on-photogram.netlify.app/

### CODE REVIEW

File: src/Components/Router.jsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+import { Home, Explore } from "../pages";
+
+const Router = () => {
+  return (
+    <Routes>
+      <Route path="/" element={<Home />} />
+      <Route path="/explore" element={<Explore />} />
+      {/* <Route path="/liked" element={<Like />} />
+      <Route path="/profile" element={<Profile />} />
+      <Route path="*" element={<PageNotFound />} /> */}
```
Comment (shivanigangadharan) [inline src/Components/Router.jsx:11]: can remove comments if not reqd and can also ommit the use of `/` as it's not reqd in react router v6.

File: src/pages/Home/home.css [hunk 2]
```diff
@@ -0,0 +1,30 @@
+.home__container {
+  /* grid-template-columns: 15% 70% 15%; */
```
Comment (shivanigangadharan) [inline src/pages/Home/home.css:2]: can remove comment here too

**Review by shivanigangadharan — APPROVED** [review]
Code looks well structured.. nice work on the UI as well!

**Review by KanekarSnehal — APPROVED** [review]
UI looks great
Code looks good to merge

File: src/Components/Router.jsx (hunk 1 above)
Comment (mnath30) [reply src/Components/Router.jsx:11]: Thankyou for your comments will make the required changes and commit in next pr

**Pull request merged**